FlowCrypt/src/main/java/com/flowcrypt/email/api/email/EmailUtil.kt

```diff
@@ -1084,6 +1058,16 @@ class EmailUtil {
       return reply
     }
 
+    /**
+     * Get public email domains.
+     */
+    fun getPublicEmailDomains(): Array<String> {
+      return arrayOf(
+        JavaEmailConstants.EMAIL_PROVIDER_GMAIL,
+        JavaEmailConstants.EMAIL_PROVIDER_GOOGLEMAIL
+      )
```

Comment:
On browser extension we use: `['gmail.com', 'yahoo.com', 'outlook.com', 'live.com', 'googlemail.com']`